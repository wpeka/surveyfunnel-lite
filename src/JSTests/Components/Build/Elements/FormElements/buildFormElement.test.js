import React from 'react';
import { shallow } from 'enzyme';
import { fireEvent, render } from '@testing-library/react';
import { BuildFormElement } from '../../../../../Components/Build/Elements/FormElements/BuildFormElement';

const component = render(<BuildFormElement />);

function dragAndDrop(knight: Element) {
    fireEvent.dragStart(knight)
    fireEvent.dragEnter(cell)
  }
  dragAndDrop(
    component.getByText('drag me'),
 );

 describe("BuildFormElement",()=>{
     
 })


