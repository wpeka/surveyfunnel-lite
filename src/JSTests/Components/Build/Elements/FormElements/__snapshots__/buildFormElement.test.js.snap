// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form elements test 1`] = `
Array [
  " ",
  <div
    className="surveyfunnel-lite-form-elements_box"
    data-testid="buildelement-"
    role="BuildElement"
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <img
      className="surveyfunnel-lite-build-simple-img"
      src="test-file-stub"
    />
    <p>
      sample
    </p>
  </div>,
  " ",
]
`;
